.github/workflows/latest-dev-fluvio.yaml

```diff
@@ -30,8 +30,7 @@ jobs:
         run: |
           # work around cargo bug
           #cargo add fluvio --git https://github.com/infinyon/fluvio --branch master
-          sed -i "s/fluvio = { features = .*/fluvio = { features = \[\"admin\"\], git = \"https:\/\/github.com\/infinyon\/fluvio.git\" }/g" Cargo.toml
-
+          sed -i.bak "s/tag = "\(.*\)"/branch = "master"/g" Cargo.toml
```

Comment:
Wondering if this is too general a replacement, but if fluvio deps are the only ones to use git tag/branch it should be fine.   Also wondering `cargo add` is fixed to update git repos dep links again.  But let's go ahead on this, latest-dev-fluvio is a noncritical workflow.

Reply:
I think that it is too general, but as you said we only use git tag right now for fluvio dependencies